buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -761,8 +761,8 @@ class BeamModulePlugin implements Plugin<Project> {
         }
         project.tasks.withType(FindBugs) {
           reports {
-            html.enabled = true
-            xml.enabled = false
+            html.enabled = false
```

Comment:
@adude3141 Do you think we could re-enable html output in addition to xml? It is helpful for looking at the findbugs results locally.

Reply:
I discovered it only supports one output.

Reply:
@kennknowles already pushed on that on https://github.com/apache/beam/pull/7704

We should definitely change that and toggle based on environment. I ll prepare something.

Reply:
Great!